test/SadTest.cc

```diff
@@ -142,6 +184,88 @@ class SADTestBase : public ::testing::Test {
         }
     }
 
+    void prepare_sad_data() {
+        const int32_t mask = (1 << 8) - 1;
+        SVTRandom rnd(0, mask);
+        switch (test_sad_pattern_) {
+        case BUF_MAX: {
+            for (int i = 0; i < 16; i++) {
```

Comment:
Replace this const initialization using a function?

Reply:
We have new variable params sad8x8 sad16x16 sad32x32 In check_get_32x32_sad and check_get_nsq_sad which are different from what in check_get_8x8_sad,so I write different new funcs to generate those params. Or you prefer to let me use same fun as check_get_8x8_sad by adding new params in  check_get_32x32_sad, check_get_nsq_sad, check_get_Extsad8x8 check_get_8x8_sad to control their params generating?